src/main/resources/changesets/execution_backend_info.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet author="mcovarr" id="CREATE_EXECUTION_INFO">
+        <createTable tableName="EXECUTION_INFO">
```

Comment:
The engine would not need to know the specific keys for a backend.  It would either receive a set of generic KV pairs from a Backend to be persisted, or load all the KV pairs and serve them back to a caller (e.g. the metadata endpoint).